plugins/profile.py

```diff
@@ -91,6 +91,21 @@ async def setprofile(msg: Message) -> None:
         await msg.conn.main_room.send_rankhtmlbox("%", message)
 
 
+@command_wrapper(
+    aliases=("clearprofilo", "resetprofile", "resetprofilo"),
+    helpstr="Rimuovi la frase personalizzata dal profilo.",
+)
+async def clearprofile(msg: Message) -> None:
+    db = Database.open()
+    with db.get_session() as session:
+        userid = msg.user.userid
+        session.add(d.Users(userid=userid))
+        query_ = session.query(d.Users).filter_by(userid=userid)
+        query_.update({"description": "", "description_pending": ""})
```

Comment:
One last thing: I would get rid of `query_` and do everything together. In `.setprofile` it's needed because of the auto-approval for staff members, but here it doesn't really make sense.
```suggestion
        session.query(d.Users).filter_by(userid=userid).update(
            {"description": "", "description_pending": ""}
        )
```